files/fr/web/svg/element/radialgradient/index.md

```diff
@@ -79,7 +79,7 @@ html,body,svg { height:100% }
 - [`gradientUnits`](/fr/docs/Web/SVG/Attribute/gradientUnits)
   - : Cet attribut définit le système de coordonnées pour les attributs `cx`, `cy`, `r`, `fx`, `fy`, et `fr`.
     - Type de valeur
-      - : `userSpaceOnUse`|`objectBoundingBox`
+      - : `userSpaceOnUse` | `objectBoundingBox`
```

Comment:
Will this be fixed on the [English page](https://github.com/mdn/content/blob/main/files/en-us/web/svg/element/radialgradient/index.md?plain=1#L65) as well?

Reply:
Yeah, I'm planning to PR some stuff upstream as well, since I relalized there are probably similar things from the conversion. I'm good reverting this one till later as I didn't change all these occurrences yet

Reply:
Ended up removing it for now when rebasing out the other file